ckan/public/base/javascript/modules/autocomplete.js

```diff
@@ -186,15 +186,20 @@ this.ckan.module('autocomplete', function (jQuery) {
      *
      * Returns a text string.
      */
-    formatResult: function (state, container, query) {
-      var term = this._lastTerm || null; // same as query.term
+    formatResult: function (state, container, query, escapeMarkup) {
+      var term = this._lastTerm || (query ? query.term : null) || null; // same as query.term
 
       if (container) {
         // Append the select id to the element for styling.
         container.attr('data-value', state.id);
```

Comment:
"term" is the query text you entered.
In the case of the this._lasterm variable, it will be updated when the query is executed as an ajax request (for example, autocomplete on an add memeber page).

However, in case of autocomplete of add group page, All option values are updated before page rendering, not ajax request.
As a result, the query text you entered is updated in the "query.term" variable.

If you test without the patch, you will see that the query text is not bolded.(on add group page)